static/usr/lib/udev/rules.d/71-nvidia.rules

```diff
@@ -1,11 +1,30 @@
+## This is an adapted version of 71-nvidia.rules file shipped by nvidia-kernel-common-*
```

Comment:
I think it would be nice to say here what is adapted. Just to make it easier to update.

Reply:
I see the path to nvidia-create-device is different. And  nvidia-persistenced is removed.

Why is nvidia-persistenced removed?

Reply:
I suppose for the persistenced,  there is no way for it to be shipped. But since it is is just a `ENV{SYSTEMD_WANTS}`, I am not sure this would cause any error.

Reply:
I've commented on the things we change.

Reply:
Thank you.